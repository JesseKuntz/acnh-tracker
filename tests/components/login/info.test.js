import { h } from 'preact';
import LoginInfo from '../../../src/components/login/info';

import { shallow } from 'enzyme';

describe('LoginInfo', () => {
  test('matches snapshot', () => {
    expect(shallow(<LoginInfo />)).toMatchSnapshot();
  });
});
