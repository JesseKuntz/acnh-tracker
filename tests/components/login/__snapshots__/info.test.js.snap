// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginInfo matches snapshot 1`] = `
<div
  className="info-container"
>
  <LoginButton />
</div>
`;
